PWGCF/TwoParticleCorrelations/DataModel/TwoParticleCorrelationsSkimmed.h

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "Framework/AnalysisTask.h"
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/ASoAHelpers.h"
+#include "Common/DataModel/TrackSelectionTables.h"
+#include "Common/DataModel/Centrality.h"
+#include "PWGCF/TwoParticleCorrelations/Core/SkimmingConfigurableCuts.h"
+#include "PWGCF/TwoParticleCorrelations/Core/EventSelectionFilterAndAnalysis.h"
+#include "PWGCF/TwoParticleCorrelations/Core/TrackSelectionFilterAndAnalysis.h"
+#include "PWGCF/TwoParticleCorrelations/Core/PIDSelectionFilterAndAnalysis.h"
+#include "Framework/runDataProcessing.h"
+
+/* TODO: this will go to its proper header at some point */
+#ifndef O2_ANALYSIS_TWOPSKIMMED_H
+#define O2_ANALYSIS_TWOPSKIMMED_H
+
+#include "Framework/ASoA.h"
+#include "Framework/AnalysisDataModel.h"
+
+namespace o2
+{
+namespace aod
+{
+/* we have to change from int to bool when bool columns work properly */
```

Comment:
I thought they are ok now. Aren't they?

Reply:
This is a legacy comment which does not apply here and which I will remove
But the current implementation of the dptdpt task still works without `bool` because last time I test them, by the end of January, they didn't work

Reply:
Yes, you are right. We should make a test case for this which we keep... (it didn't group at that time, right?)